coredns/tox.ini

```diff
@@ -0,0 +1,24 @@
+[tox]
+minversion = 2.0
+basepython = py27
+envlist = unit, flake8
+
+[testenv]
+platform = linux2|darwin
+deps =
+    datadog-checks-base
```

Comment:
can you change this to `-e../datadog_checks_base[deps]`? That will, I believe, make the test work.